scripts/release/pipeline_main.py

```diff
@@ -67,6 +69,7 @@ def get_builder_function_for_image_name() -> Dict[str, Callable]:
     image_builders = {
         MEKO_TESTS_IMAGE: build_tests_image,
         OPERATOR_IMAGE: build_operator_image,
+        OPERATOR_RACE_IMAGE: partial(build_operator_image, with_race_detection=True),
```

Comment:
nit: Do you think this deserves a comment ? I think most people who will read this in the future won't know what `partial` is

Reply:
I'd rather not add a comment to document partial function application that's been in the python stdlib forever, but if we consider it necessary I will. Something like "Use the same image builder as the operator image with defaults set to enable race detection" ?